### ISSUE TITLE
ethereum wallet not synching

### ISSUE THREAD

**Issue by doctorbhoo** [ethereum/mist#3526]
I can't start ethereum node!! keep getting message stating that my computer clock is not synched. The clock looks like it is synched,it synchs automatically!

- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```

I can't start ethereum node!! keep getting message stating that my computer clock is not synched. The clock looks like it is synched,it synchs automatically!

**Comment by evertonfraga** [comment]
Hi @doctorbhoo,

what OS do you use? Do you have advanced NTP clock sync?
Please also try

**Comment by doctorbhoo** [comment]
Hi,
I use macbook and the clock is set to sync automatically




Prakash S. Bhoopalam.



2525 W University Avenue #404 | Muncie, IN 47303

Phone:   765 231 9494  |  E  doctorbhoo@hotmail.com

WWW.PAVILIONPEDIATRICCENTER.COM
________________________________

**Comment by evertonfraga** [comment]
A fix is on the way.

**Comment by evertonfraga** [comment]
Fixed in 0.10.0. Please update.
